ydb/core/mind/hive/monitoring.cpp

```diff
@@ -2427,7 +2427,10 @@ class TTxMonEvent_LandingData : public TTransactionBase<THive> {
                 TNodeInfo& node = *nodeInfo;
                 TNodeId id = node.Id;
 
-                if (!node.IsAlive() && TInstant::MilliSeconds(node.Statistics.GetLastAliveTimestamp()) < aliveLine) {
+                if (!node.IsAlive()
```

Comment:
it means that node will disappear on changing any of these flags. 
it would look strange... maybe consider removing this check at all?